charts/signoz/templates/otel-collector-metrics/deployment.yaml

```
@@ -111,6 +111,9 @@ spec:
           volumeMounts:
             - name: otel-collector-metrics-config-vol
               mountPath: /conf
+              {{- if .Values.otelCollectorMetrics.extraVolumeMounts }}
+              {{- toYaml .Values.otelCollectorMetrics.extraVolumeMounts | nindent 12 }}
+              {{- end }}
```

minor indentation pattern suggestion

```suggestion
            {{- if .Values.otelCollectorMetrics.extraVolumeMounts }}
              {{- toYaml .Values.otelCollectorMetrics.extraVolumeMounts | nindent 12 }}
            {{- end }}
```